### ISSUE TITLE
Issue Callback ALM Empty

### ISSUE THREAD

**Issue by manuhook** [dcooney/wordpress-ajax-load-more#120]
Hi,
$.fn.almEmpty is not firing.

I need to update the code like that in order to make it work : 
 
```
// ALM Empty
if((total === 0) || (total == null)){
               if ($.isFunction($.fn.almEmpty)) {
                  $.fn.almEmpty(alm);
               }
            }
```

When i have no post the ajax request is answering : 

`{"html":null,"meta":{"postcount":null,"totalposts":null,"debug":null}}`

**Comment by dcooney** [comment]
This was fixed in 2.13.0.1 - https://wordpress.org/plugins/ajax-load-more/

I have not pushed the changes to github.

**Comment by manuhook** [comment]
Oh ok thank you ;)
